lms/static/sass/base/_base.scss

```diff
@@ -122,6 +122,10 @@ a:focus {
   margin: 0 auto;
   background: $content-wrapper-bg;
   padding-bottom: ($baseline*2);
+
+  @media print {
```

Comment:
This removes 40px of whitespace from the bottom of the page. This whitespace looks good on screen, but can force a new page to be printed (even if it has nothing on it).